episodes/24-collaborating.md

```diff
@@ -37,21 +57,68 @@ you may also want to assign these issues to a member of your lesson development
+
+GitHub provides template boards that by default contain the three ‘basic’ columns, with pretty self-explanatory names:
+
+- `To Do`
+- `In Progress`
+- `Done`
+
+If you add an issue or pull request to a card in the board, it will be automatically be moved it to ‘Done’ for you when you close the issue or merge the pull request. One common extra column is `On hold` or `Waiting`. If you have tasks that get held up by waiting on other people (e.g. to respond to your questions) then moving them to a separate column makes their current state clearer. 
+
+You can also create a card without an issue. Such cards (or notes) can have detailed content like checklists. GitHub also allows you to convert a card to an issue so you can add labels or detailed comments to it. Sometimes, a card you thought was simple and self-contained might turn out to be a bigger task than you anticipated - in that case, it is sensible to create new cards that reference the one they broke off from.
+
+Once your project board has a large number of cards on it, you might want to begin priorisiting them. Not all tasks are going to be equally important, and some will require others to be completed before they can even be begun. Common methods of prioritisation include:
+
+- **Vertical position**: the vertical arrangement of cards in a column implicitly represents their importance. High-priority bugs go to the top of `To Do`, whilst tasks that depend on others go beneath them. This is the easiest one to implement, though you have to remember to correctly place cards when you add them.
+- **Priority columns**: instead of a single `To Do` column, you can have two or more, for example - `To Do: Low Priority` and `To Do: High Priority`. When adding a card, you pick which is the appropriate column for it. You can even add a Triage column for newly-added issues that you’ve not yet had time to classify. This format works well for project boards devoted to bugs.
+- **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and setting their colours. Green low, orange medium and red high priority labels make for a very visually clear indication, but require the most admin work as each card has to be an issue to be assigned a label.
```

Comment:
```suggestion
- **Labels**: if you convert each card into an issue, then you can label them with their priority - remember GitHub lets you create custom labels and set their colours. Green for low, orange for medium and red for high priority labels make for a very visually clear indication, but require the most admin work as each card has to be an issue to be assigned a label.
```

Reply:
> make for a very visually clear indication

Perhaps not for people with colour blindness?

Reply:
Maybe we should leave out suggestions as to what colours use to indicate the priority - or mention the traffic light system but warn about it accessibility for colour blind users.